modules/op-signing-secrets-in-tekton-chains.adoc

```diff
@@ -0,0 +1,60 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-chains-for-pipelines-supply-chain-security.adoc
+
+:_content-type: CONCEPT
+[id="signing-secrets-in-tekton-chains_{context}"]
+= Signing secrets in Tekton Chains
+
+Cluster administrators can generate a key pair and use Chains to sign artifacts using a Kubernetes secret. For Chains to work, a private key and a password for encrypted keys must exist as part of the `signing-secrets` Kubernetes secret, in the `tekton-chains` namespace.
+
+Currently, Chains supports the `x509` and `cosign` signature schemes.
+
+[NOTE]
+====
+Use only one of the supported signature schemes.
+====
+
+[id="chains-signing-secrets-x509_{context}"]
+== Signing using x509
+
+To use the `x509` signing scheme with Chains, store the `x509.pem` private key of the `ed25519` or `ecdsa` type in the `signing-secrets` Kubernetes secret. Ensure that the key is stored as an unencrypted PKCS8 PEM file (`BEGIN PRIVATE KEY`).
+
+[id="chains-signing-secrets-cosign_{context}"]
+== Signing using cosign
```

Comment:
Give the procedure the heading, `.Procedure` like you do in your other procedures. 

So:

.Procedure 

1. Install cosign. 
2. so on and so forth

Reply:
See my other similar counterarguments.